shared/styles/grid.scss

```diff
@@ -39,10 +39,31 @@
 }
 
 // Offsets
-@mixin make-offset-left($offset: percentage(1/12)) {
+@mixin grid-offset-left($offset: percentage(1/$grid-column-count)) {
   margin-left: convertify($offset);
 }
 
-@mixin make-offset-right($offset: percentage(1/12)) {
+@mixin grid-offset-right($offset: percentage(1/$grid-column-count)) {
```

Comment:
Could the default `$offset:` just be `1` on lines `42` and `46`? Because the `percentage($number / $grid-column-count)` is already getting applied via `convertify($offset)`?

Reply:
Yeah, since that's the expected behaviour when you're using it